h2/utilities.py

```diff
@@ -329,6 +329,16 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
             seen_pseudo_header_fields
         )
 
+    # If ':status' pseudo-header is not there in a response header, reject it
+    if (
+        hdr_validation_flags.is_response_header and
+        b':status' not in seen_pseudo_header_fields and
+        u':status' not in seen_pseudo_header_fields
+    ):
```

Comment:
Rather than have this gigantic if statement with confusing indentation, can we save the boolean value into a temporary variable and then `if` on that?